src/source/reference/post-metrics-data.rst

```diff
@@ -4,6 +4,23 @@
 POST /metrics/data
 =======================
 
+V2
+~~~
+.. list-table::
+ :widths: 20 80
+
+ * - Scope
+   - ``console_ro`` or ``api_ro``
+
+ * - Accept
+   - ``application/json``
+ * - Content-Type
+   - ``application/sellside.metrics.data-v2+json``
+
+Version 2 is replacing fields that are showing values in cents with their corresponding values in micros. One cent equals to 10000 micros. This allows more granular setting of bid values which was limited down to the size of one cent. This version also introduces spent vs bid values differentiation which is expressed by BidMicros and SpentMicros. 
```

Comment:
See my comment above. Have a file with the general explanation of micros/bidding, import or link the file (whichever works best) in all endpoints affected.

Reply:
`size of one cent` what :D  cents have size?

Reply:
Yep what i ment to capture is the size of 100 as part of euro, fraction maybe a better choice, what do you recommend ?